src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -345,6 +345,13 @@ internal int AssertCount
         /// </summary>
         public int TestCaseTimeout { get; set; }
 
+#if !THREAD_ABORT
+        /// <summary>
+        /// Gets or sets the <see cref="CancellationToken"/> for the test case.
+        /// </summary>
+        public CancellationToken CancellationToken { get; internal set; } = CancellationToken.None;
```

Comment:
Similar to @Dreamescaper 's question above, would this mean we'd have a different API shape for netstandard + netfx builds of the framework here and in `TestContext` below?  Or would we want to expose this in all builds and just have it be a (temporary?) no-op when run in a netfx context?